devtools/prod-envs/qcarchive-worker-openff-psi4-basis-set-update.yaml

```diff
@@ -0,0 +1,23 @@
+name: qcarchive-worker-openff-psi4
+channels:
+  - conda-forge/label/libint_dev
+  - conda-forge
+dependencies:
+  - python =3.10
```

Comment:
(not blocking) We'll want to stop pinning to python 3.10 for our images in the next few months, since it's falling out of the SPEC0 range. I'll open an issue to track this.

Reply:
I've been using 3.11 locally so I have confidence that that the runners will be fine with that. I'll update it to that.